gcc-4.8/build.sh

```diff
@@ -35,7 +35,9 @@ else
     # For reference during post-link.sh, record some
     # details about the OS this binary was produced with.
     mkdir -p "${PREFIX}/share"
-    cat /etc/*-release > "${PREFIX}/share/conda-gcc-build-machine-os-details"
+	# lsb_release can complain about LSB mosules in stderr, so we
+	# ignore that.
+    lsb_release -a 1> "${PREFIX}/share/conda-gcc-build-machine-os-details"
```

Comment:
>I'd say that this seems a reasonable solution unless willing to do something much more complex.

Agreed.  Even if few distros are "LSB Certified", it looks like most distros do support the `lsb_release` command.